data/csp/gce/ondemand/config.yaml

```diff
@@ -0,0 +1,4 @@
+config:
+  services:
+    cloudreg:
+      - guestregister
```

Comment:
While this would work, we decided at some point to completely separate SLE 12 and 15 configurations. It creates duplication but keeps it simpler overall and avoids accidentally changing something in the other code stream. So this should be in `sle12` and `sle15` sub dirs under `ondemand`.